tests/core/test_rule_groups.py

```diff
@@ -0,0 +1,38 @@
+from smdebug_rulesconfig.debugger_rules._constants import (
+    SUPPORTED_DL_FRAMEWORKS,
+    SUPPORTED_FRAMEWORKS,
+)
+from smdebug_rulesconfig.debugger_rules._ruleGroups import (
+    DEEP_LEARNING_APPLICATION_RULES,
+    DEEP_LEARNING_RULES,
+    UNIVERSAL_RULES,
+    XGBOOST_RULES,
+)
+from smdebug_rulesconfig.debugger_rules._utils import _get_rule_list
+
+
+def test_universal_rules():
+    for rule in UNIVERSAL_RULES:
+        for framework in SUPPORTED_FRAMEWORKS:
```

Comment:
seems like a good use case for pytest.mark.parametrize if you're simply iterating across each combination of rule and framework.

alternatively, you can aggregate all of the tests into one and just paremetrize on the list of rules and frameworks to reduce the amount of duplicated code.

Reply:
Done.